### PR TITLE
Conan Exiles: various updates

### PR DESCRIPTION
**Pull request by Greelan**
Switch to new port system, add Wine env vars, change AdminPassword setting to be user configurable, other tweaks

### CODE REVIEW

**Review by IceOfWraith — DISMISSED** [review]
Good stuff! Keep them coming!

**Review by IceOfWraith — APPROVED** [review]

**Comment by Greelan** [comment]
Now updated to set the AdminPassword via ini command line param. As a result the config download is no longer needed (keeping the config in the repo for legacy installations)

**Comment by Greelan** [comment]
Switching back from using new ports system, given the Refresh Configuration logic may not be able to handle it

**Comment by Greelan** [comment]
Reverted ports switch, now it has the new ports system again

File: conan-exilesconfig.json [hunk 1]
```diff
@@ -161,7 +158,7 @@
         "IncludeInCommandLine":false,
         "DefaultValue":"0",
         "EnumValues":{
-            "0":"Europe",
+            "0":"Europe (default)",
```
Comment (IceOfWraith) [inline conan-exilesconfig.json:161]: You knew I'd ask this. Is there a benefit to calling out a default when it is different for all users?

**Review by IceOfWraith — APPROVED** [review]
All looks good other than the one question

File: conan-exilesconfig.json (hunk 1 above)
Comment (Greelan) [reply conan-exilesconfig.json]: Fixed

**Review by IceOfWraith — APPROVED** [review]

**Pull request merged**